### ISSUE TITLE
6.0.7 startet nicht 

### ISSUE THREAD

**Issue by Brainbug01** [ioBroker/ioBroker.js-controller#2837]
### No existing issues.

- [X] There is no existing issue for my problem.

### Description

```
2024-07-13 09:59:01.727  - info: host.iobroker iobroker.js-controller version 6.0.7 js-controller starting
2024-07-13 09:59:01.729  - info: host.iobroker Copyright (c) 2014-2023 bluefox, 2014 hobbyquaker
2024-07-13 09:59:01.729  - info: host.iobroker hostname: iobroker, node: v20.15.1
2024-07-13 09:59:01.730  - info: host.iobroker ip addresses: 192.168.0.222 fe80::8c62:1cff:fe49:dd99
2024-07-13 09:59:01.779  - info: host.iobroker connected to Objects and States
2024-07-13 09:59:01.787  - info: host.iobroker added notifications configuration of host
2024-07-13 09:59:01.901  - info: host.iobroker 62 instances found
2024-07-13 09:59:01.919  - info: host.iobroker starting 51 instances
2024-07-13 09:59:02.046  - info: host.iobroker instance system.adapter.admin.0 in version "7.0.13" started with pid 2920
2024-07-13 09:59:02.053  - info: host.iobroker Delete state "system.host.iobroker.versions.nodeNewestNext"
2024-07-13 09:59:02.055  - info: host.iobroker Delete state "system.host.iobroker.versions.npmCurrent"
2024-07-13 09:59:02.056  - info: host.iobroker Delete state "system.host.iobroker.versions.nodeCurrent"
2024-07-13 09:59:02.057  - info: host.iobroker Delete state "system.host.iobroker.versions.npmNewestNext"
2024-07-13 09:59:02.058  - info: host.iobroker Delete state "system.host.iobroker.versions.npmNewest"
2024-07-13 09:59:02.059  - info: host.iobroker Delete state "system.host.iobroker.versions.nodeNewest"
2024-07-13 09:59:02.060  - info: host.iobroker Some obsolete host states deleted.
2024-07-13 09:59:02.745  - error: admin.0 (2920) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-13 09:59:02.745  - error: admin.0 (2920) unhandled promise rejection: callback is not a function
2024-07-13 09:59:02.746  - error: admin.0 (2920) TypeError: callback is not a function
    at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
    at runNextTicks (node:internal/process/task_queues:60:5)
    at processImmediate (node:internal/timers:449:9)
2024-07-13 09:59:02.747  - error: admin.0 (2920) callback is not a function
2024-07-13 09:59:03.253  - warn: admin.0 (2920) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-13 09:59:03.782  - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-13 09:59:03.782  - error: host.iobroker Caught by controller[1]: TypeError: callback is not a function
2024-07-13 09:59:03.782  - error: host.iobroker Caught by controller[1]:     at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:03.782  - error: host.iobroker Caught by controller[1]:     at runNextTicks (node:internal/process/task_queues:60:5)
2024-07-13 09:59:03.783  - error: host.iobroker Caught by controller[1]:     at processImmediate (node:internal/timers:449:9)
2024-07-13 09:59:03.783  - error: host.iobroker instance system.adapter.admin.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-13 09:59:03.783  - info: host.iobroker Restart adapter system.adapter.admin.0 because enabled
2024-07-13 09:59:06.013  - info: host.iobroker instance system.adapter.email.0 in version "1.3.1" started with pid 2935
2024-07-13 09:59:06.457  - error: email.0 (2935) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-13 09:59:06.458  - error: email.0 (2935) unhandled promise rejection: callback is not a function
2024-07-13 09:59:06.459  - error: email.0 (2935) TypeError: callback is not a function
    at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:06.459  - error: email.0 (2935) callback is not a function
2024-07-13 09:59:06.967  - warn: email.0 (2935) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-13 09:59:07.492  - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-13 09:59:07.493  - error: host.iobroker Caught by controller[1]: TypeError: callback is not a function
2024-07-13 09:59:07.493  - error: host.iobroker Caught by controller[1]:     at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:07.493  - error: host.iobroker instance system.adapter.email.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-13 09:59:07.493  - info: host.iobroker Restart adapter system.adapter.email.0 because enabled
2024-07-13 09:59:10.014  - info: host.iobroker instance system.adapter.javascript.0 in version "8.6.0" started with pid 2964
2024-07-13 09:59:10.732  - error: javascript.0 (2964) An error happened which is most likely from one of your scripts, but the originating script could not be detected.
2024-07-13 09:59:10.733  - error: javascript.0 (2964) Error: callback is not a function
2024-07-13 09:59:10.733  - error: javascript.0 (2964) TypeError: callback is not a function
    at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
    at runNextTicks (node:internal/process/task_queues:60:5)
    at processImmediate (node:internal/timers:449:9)
2024-07-13 09:59:14.009  - info: host.iobroker instance system.adapter.pushover.0 in version "3.0.6" started with pid 2979
2024-07-13 09:59:14.454  - error: pushover.0 (2979) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-13 09:59:14.455  - error: pushover.0 (2979) unhandled promise rejection: callback is not a function
2024-07-13 09:59:14.456  - error: pushover.0 (2979) TypeError: callback is not a function
    at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:14.456  - error: pushover.0 (2979) callback is not a function
2024-07-13 09:59:14.965  - warn: pushover.0 (2979) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-13 09:59:15.490  - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-13 09:59:15.490  - error: host.iobroker Caught by controller[1]: TypeError: callback is not a function
2024-07-13 09:59:15.490  - error: host.iobroker Caught by controller[1]:     at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:15.490  - error: host.iobroker instance system.adapter.pushover.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-13 09:59:15.490  - info: host.iobroker Restart adapter system.adapter.pushover.0 because enabled
2024-07-13 09:59:18.033  - info: host.iobroker instance system.adapter.pushover.1 in version "3.0.6" started with pid 2994
2024-07-13 09:59:18.473  - error: pushover.1 (2994) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-13 09:59:18.474  - error: pushover.1 (2994) unhandled promise rejection: callback is not a function
2024-07-13 09:59:18.476  - error: pushover.1 (2994) TypeError: callback is not a function
    at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
    at runNextTicks (node:internal/process/task_queues:60:5)
    at processImmediate (node:internal/timers:449:9)
2024-07-13 09:59:18.476  - error: pushover.1 (2994) callback is not a function
2024-07-13 09:59:18.984  - warn: pushover.1 (2994) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-13 09:59:19.509  - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-13 09:59:19.509  - error: host.iobroker Caught by controller[1]: TypeError: callback is not a function
2024-07-13 09:59:19.509  - error: host.iobroker Caught by controller[1]:     at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:19.509  - error: host.iobroker Caught by controller[1]:     at runNextTicks (node:internal/process/task_queues:60:5)
2024-07-13 09:59:19.509  - error: host.iobroker Caught by controller[1]:     at processImmediate (node:internal/timers:449:9)
2024-07-13 09:59:19.509  - error: host.iobroker instance system.adapter.pushover.1 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-13 09:59:19.510  - info: host.iobroker Restart adapter system.adapter.pushover.1 because enabled
2024-07-13 09:59:22.012  - info: host.iobroker instance system.adapter.pushover.2 in version "3.0.6" started with pid 3009
2024-07-13 09:59:22.451  - error: pushover.2 (3009) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-13 09:59:22.452  - error: pushover.2 (3009) unhandled promise rejection: callback is not a function
2024-07-13 09:59:22.453  - error: pushover.2 (3009) TypeError: callback is not a function
    at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:22.454  - error: pushover.2 (3009) callback is not a function
2024-07-13 09:59:22.503  - info: pushover.2 (3009) starting. Version 3.0.6 in /opt/iobroker/node_modules/iobroker.pushover, node: v20.15.1, js-controller: 6.0.7
2024-07-13 09:59:22.960  - info: pushover.2 (3009) terminating
2024-07-13 09:59:22.961  - warn: pushover.2 (3009) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-13 09:59:23.487  - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-13 09:59:23.487  - error: host.iobroker Caught by controller[1]: TypeError: callback is not a function
2024-07-13 09:59:23.487  - error: host.iobroker Caught by controller[1]:     at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:23.487  - error: host.iobroker instance system.adapter.pushover.2 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-13 09:59:23.487  - info: host.iobroker Restart adapter system.adapter.pushover.2 because enabled
2024-07-13 09:59:26.012  - info: host.iobroker instance system.adapter.telegram.0 in version "3.7.1" started with pid 3024
2024-07-13 09:59:26.609  - error: telegram.0 (3024) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-13 09:59:26.610  - error: telegram.0 (3024) unhandled promise rejection: callback is not a function
2024-07-13 09:59:26.611  - error: telegram.0 (3024) TypeError: callback is not a function
    at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:26.612  - error: telegram.0 (3024) callback is not a function
2024-07-13 09:59:27.118  - warn: telegram.0 (3024) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-13 09:59:27.645  - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-13 09:59:27.645  - error: host.iobroker Caught by controller[1]: TypeError: callback is not a function
2024-07-13 09:59:27.645  - error: host.iobroker Caught by controller[1]:     at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:27.645  - error: host.iobroker instance system.adapter.telegram.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-13 09:59:27.645  - info: host.iobroker Restart adapter system.adapter.telegram.0 because enabled
2024-07-13 09:59:28.745  - info: host.iobroker instance system.adapter.telegram.0 in version "3.7.1" started with pid 3039
2024-07-13 09:59:29.351  - error: telegram.0 (3039) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-13 09:59:29.351  - error: telegram.0 (3039) unhandled promise rejection: callback is not a function
2024-07-13 09:59:29.353  - error: telegram.0 (3039) TypeError: callback is not a function
    at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
    at runNextTicks (node:internal/process/task_queues:60:5)
    at processImmediate (node:internal/timers:449:9)
2024-07-13 09:59:29.353  - error: telegram.0 (3039) callback is not a function
2024-07-13 09:59:29.861  - warn: telegram.0 (3039) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-13 09:59:29.999  - info: host.iobroker instance system.adapter.telegram.1 in version "3.7.1" started with pid 3054
2024-07-13 09:59:30.391  - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-13 09:59:30.391  - error: host.iobroker Caught by controller[1]: TypeError: callback is not a function
2024-07-13 09:59:30.391  - error: host.iobroker Caught by controller[1]:     at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:30.391  - error: host.iobroker Caught by controller[1]:     at runNextTicks (node:internal/process/task_queues:60:5)
2024-07-13 09:59:30.391  - error: host.iobroker Caught by controller[1]:     at processImmediate (node:internal/timers:449:9)
2024-07-13 09:59:30.391  - error: host.iobroker instance system.adapter.telegram.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-13 09:59:30.391  - info: host.iobroker Restart adapter system.adapter.telegram.0 because enabled
2024-07-13 09:59:30.593  - error: telegram.1 (3054) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-13 09:59:30.593  - error: telegram.1 (3054) unhandled promise rejection: callback is not a function
2024-07-13 09:59:30.595  - error: telegram.1 (3054) TypeError: callback is not a function
    at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
    at runNextTicks (node:internal/process/task_queues:60:5)
    at processImmediate (node:internal/timers:449:9)
2024-07-13 09:59:30.595  - error: telegram.1 (3054) callback is not a function
2024-07-13 09:59:31.103  - warn: telegram.1 (3054) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-13 09:59:31.500  - info: host.iobroker instance system.adapter.telegram.0 in version "3.7.1" started with pid 3069
2024-07-13 09:59:31.632  - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-13 09:59:31.632  - error: host.iobroker Caught by controller[1]: TypeError: callback is not a function
2024-07-13 09:59:31.632  - error: host.iobroker Caught by controller[1]:     at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:31.632  - error: host.iobroker Caught by controller[1]:     at runNextTicks (node:internal/process/task_queues:60:5)
2024-07-13 09:59:31.632  - error: host.iobroker Caught by controller[1]:     at processImmediate (node:internal/timers:449:9)
2024-07-13 09:59:31.632  - error: host.iobroker instance system.adapter.telegram.1 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-13 09:59:31.632  - info: host.iobroker Restart adapter system.adapter.telegram.1 because enabled
2024-07-13 09:59:32.136  - error: telegram.0 (3069) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-13 09:59:32.136  - error: telegram.0 (3069) unhandled promise rejection: callback is not a function
2024-07-13 09:59:32.138  - error: telegram.0 (3069) TypeError: callback is not a function
    at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
    at runNextTicks (node:internal/process/task_queues:60:5)
    at processImmediate (node:internal/timers:449:9)
2024-07-13 09:59:32.138  - error: telegram.0 (3069) callback is not a function
2024-07-13 09:59:32.645  - warn: telegram.0 (3069) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-13 09:59:32.721  - info: host.iobroker instance system.adapter.telegram.1 in version "3.7.1" started with pid 3084
2024-07-13 09:59:33.172  - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-13 09:59:33.172  - error: host.iobroker Caught by controller[1]: TypeError: callback is not a function
2024-07-13 09:59:33.172  - error: host.iobroker Caught by controller[1]:     at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
2024-07-13 09:59:33.172  - error: host.iobroker Caught by controller[1]:     at runNextTicks (node:internal/process/task_queues:60:5)
2024-07-13 09:59:33.172  - error: host.iobroker Caught by controller[1]:     at processImmediate (node:internal/timers:449:9)
2024-07-13 09:59:33.173  - error: host.iobroker instance system.adapter.telegram.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-13 09:59:33.173  - info: host.iobroker Restart adapter system.adapter.telegram.0 because enabled
2024-07-13 09:59:33.173  - warn: host.iobroker Do not restart adapter system.adapter.telegram.0 because restart loop detected
2024-07-13 09:59:33.321  - error: telegram.1 (3084) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-13 09:59:33.321  - error: telegram.1 (3084) unhandled promise rejection: callback is not a function
2024-07-13 09:59:33.323  - error: telegram.1 (3084) TypeError: callback is not a function
    at /opt/iobroker/node_modules/@iobroker/plugin-base/lib/PluginBase.js:310:17
    at runNextTicks (node:internal/process/task_queues:60:5)
    at processImmediate (node:internal/timers:449:9)
2024-07-13 09:59:33.323  - error: telegram.1 (3084) callback is not a function
2024-07-13 09:59:33.831  - warn: telegram.1 (3084) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-13 09:59:33.874  - info: host.iobroker instance system.adapter.admin.0 in version "7.0.13" started with pid 3099
2024-07-13 09:59:34.004  - info: host.iobroker instance system.adapter.telegram.2 in version "3.7.1" started with pid 3106
2024-07-13 09:59:34.362  - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-13 09:59:34.362  - error: host.iobroker Caught by controller[1]: TypeError: callback is not a function
```

### JS-controller version

6.0.7

### Node version

20.15.1

**Comment by foxriver76** [comment]
Should be fixed with 6.0.8

**Comment by Brainbug01** [comment]
Funktioniert wieder.
